DockDoor/AppDelegate.swift

```diff
@@ -40,7 +41,9 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         } else {
             self.configureMenuBar()
             dockObserver = DockObserver.shared
-            keybindHelper = KeybindHelper.shared
+            if(Defaults[.showWindowSwitcher]){ // Check if the window switching functionality is enabled or not
```

Comment:
Parentheses have their place in swift, for a simple conditional like this I would just go with no parentheses. They are not needed.

Reply:
Also, comment isn't needed. I can glance at the if conditional and immediately know what it means, which means you did a good job at naming the variables :)

Readable code > comments